README.md

```diff
@@ -1,38 +1,76 @@
-# Sourcery VS Code Extension
+[twitter-shield]: https://img.shields.io/twitter/follow/SourceryAI?style=social
+[twitter-url]: https://bit.ly/sourceryai-twitter
+[github-shield]: https://img.shields.io/github/stars/sourcery-ai/sourcery?style=social
+[github-url]: https://bit.ly/sourceryai-github
+[vscode-shield]: https://img.shields.io/visual-studio-marketplace/r/sourcery.sourcery?logo=visual-studio-code&style=social
+[vscode-url]: https://bit.ly/sourceryai-vscode
 
-[Sourcery](https://sourcery.ai) is an AI-powered coding assistant which helps you write better Python code faster.
- It works by providing refactoring suggestions on the fly that you can instantly integrate into your code.
+[![Github Repo][github-shield]][github-url]
+[![VSCode Plugin][vscode-shield]][vscode-url]
+[![Twitter Follow][twitter-shield]][twitter-url]
 
-This extension is currently in beta - please raise any problems or feedback as issues.
+# Sourcery for VS Code
 
-## Installation
+Sourcery is your personal coding assistant that helps you write better, cleaner, Python code. It suggests refactorings on the fly to instantly improve your code. 
 
-1. Open VS Code and press `Ctrl+P` (`Cmd+P` on Mac) then paste in `ext install sourcery.sourcery` and press `Enter`. 
-2. Click [here](https://sourcery.ai/download/?editor=vscode) to get a free token and copy it to your clipboard.
-3. Search for `sourcery` in the VS Code settings and enter the token into the ```Sourcery Token``` field.
+![Sourcery in VS Code](https://raw.githubusercontent.com/sourcery-ai/sourcery-vscode/master/assets/VS_Code_Refactorings.gif)
 
-## How it works
+---
 
-Sourcery scans the Python file that you have open in your editor. Refactoring suggestions are 
-underlined and listed in the Problems window. 
+### Using Sourcery
 
-You can hover over a suggestion to see a diff of the changes.
+Sourcery runs in the background whenever you're working on Python files. Keep an eye out for sections of your code underlined in blue - that means Sourcery has a suggestion!
 
-To implement the suggestion use the Quick Fix from either the Problems window or the lightbulb in the code editor.
+Hover over a Sourcery suggestion to see a description of what Sourcery wants you to change along with the diff of the proposed changes. Then either use the quickfix menu (`Ctrl+` or `Cmd+` for most people) or click on the lightbulb icon to bring up the Sourcery menu and choose to accept or reject the changes.
```

Comment:
@Hellebore I think the quickfix menu needs to be updated to "(`Ctrl .` or `Cmd .` for most people)"